### PR TITLE
Adding more packages that support python3 without classifiers

### PR DESCRIPTION
**Pull request by yablonsky**
Added packages with links for convenience:
 * `avro`: https://pypi.org/project/avro-python3/
 * `confluent-kafka`: https://github.com/confluentinc/confluent-kafka-python/blob/master/tox.ini
 * `geventhttpclient`: https://github.com/gwik/geventhttpclient/blob/master/README.mdown
 * `mandrill`: https://pypi.org/project/mandrill-37/
 * `mockredispy`: https://github.com/locationlabs/mockredis/blob/master/README.md
 * `proxytypes`: https://github.com/PEAK-Legacy/ProxyTypes (see project description in GitHub)
 * `spotipy`: https://github.com/plamere/spotipy/blob/master/tox.ini

### CODE REVIEW

File: caniusepython3/overrides.json [hunk 1]
```diff
@@ -1,10 +1,12 @@
 {
+    "avro": "https://pypi.org/project/avro-python3/",
     "backport_collections": "https://docs.python.org/3/library/collections.html#module-collections",
     "backports.lzma": "https://pypi.org/project/backports.lzma/",
     "beautifulsoup": "https://pypi.python.org/pypi/beautifulsoup4",
     "bs4": "https://pypi.org/project/bs4/",
     "cheetah": "https://pypi.org/project/Cheetah3/",
     "collective.recipe.template": "https://pypi.python.org/pypi/collective.recipe.template",
+    "confluent-kafka": "https://github.com/confluentinc/confluent-kafka-python/blob/master/tox.ini",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]:
```suggestion
    "confluent-kafka": "https://pypi.org/project/confluent-kafka/#files",
```

File: caniusepython3/overrides.json [hunk 2]
```diff
@@ -21,9 +23,12 @@
     "flask-principal": "http://pythonhosted.org/Flask-Principal/changelog.html",
     "functools32": "https://pypi.python.org/pypi/functools32",
     "futures": "https://docs.python.org/3/library/concurrent.futures.html",
+    "hash-ring": "https://github.com/Doist/hash_ring/blob/master/tox.ini",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]: This actually isn't accurate as they added the test support after their latest release.
```suggestion
```

File: caniusepython3/overrides.json [hunk 3]
```diff
@@ -21,9 +23,12 @@
...
     "geoip": "https://pypi.python.org/pypi/GeoIP/",
+    "geventhttpclient": "https://github.com/gwik/geventhttpclient/blob/master/README.mdown",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]:
```suggestion
    "geventhttpclient": "https://pypi.org/project/geventhttpclient/#description",
```

File: caniusepython3/overrides.json [hunk 4]
```diff
@@ -21,9 +23,12 @@
...
     "google-apitools": "https://github.com/google/apitools/blob/master/.travis.yml",
     "grab": "https://pypi.python.org/pypi/grab",
+    "grequests": "https://github.com/kennethreitz/grequests/blob/master/setup.py",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]:
```suggestion
```
This isn't known as the `setup.py` changes came after the last release.

File: caniusepython3/overrides.json [hunk 5]
```diff
@@ -45,6 +52,7 @@
     "pandocfilters": "https://github.com/jgm/pandocfilters/blob/master/setup.py",
     "pefile": "https://github.com/erocarrera/pefile/blob/master/.travis.yml",
     "pil": "https://pypi.python.org/pypi/Pillow",
+    "proxytypes": "https://github.com/PEAK-Legacy/ProxyTypes",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]:
```suggestion
    "proxytypes": "https://pypi.org/project/ProxyTypes/#files",
```

File: caniusepython3/overrides.json [hunk 6]
```diff
@@ -69,6 +77,7 @@
     "ssh": "https://github.com/bitprophet/ssh/",
     "sklearn": "https://pypi.org/project/scikit-learn",
     "slumber": "https://pypi.org/project/slumber/",
+    "spotipy": "https://github.com/plamere/spotipy/blob/master/tox.ini",
```
Comment (brettcannon) [inline caniusepython3/overrides.json]:
```suggestion
    "spotipy": "https://github.com/plamere/spotipy",
```

**Comment by yablonsky** [comment]
Hi @brettcannon :wave: 
Thanks for your review, I applied all suggested changes.

**Comment by brettcannon** [comment]
Thanks!

**Pull request merged**